lib/src/exception/name_exception.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:rohd/rohd.dart';
```

Comment:
refer to ``modules.dart`` for export

Reply:
Add ``exceptions.dart`` but would prefer to only import specific exception in test to follow minimal to use rule of thumb. This method is still open to discussion.

Reply:
That's a fair point.  The question is whether we would want to expose all the Exceptions whenever someone imports ROHD's package, and I think usually they probably don't since they won't need to throw their own versions of most of the exceptions.  What you have here should be good I think.